shared/constants/fs.js

```diff
@@ -503,13 +503,14 @@ export const userTlfHistoryRPCToState = (
   return I.List(updates)
 }
 
+const supportedImgMimeTypes = ['image/png', 'image/jpeg', 'image/gif', 'image/webp']
```

Comment:
`new Set(['image/png, ...])`
...
`supportedImgMimeTypes.has(mimeType)`

Reply:
OK. Although I actually thought about that, but there are four items here, and I'm not sure having that would actually be faster.

Reply:
There are 4 now, but there will likely be more over time.